scripts/deploy.sh

```diff
@@ -127,6 +128,12 @@ if [[ "${DEPLOY_OA}" == "yes" ]]; then
     run_ansible haproxy-install.yml
   fi
 
+  # Apply security hardening
+  if [[ "$DEPLOY_HARDENING" == "yes" ]]
+  then
```

Comment:
Can we set this to false if $DEPLOY_HARDENING != "yes"? Just so no one gets caught out.